docs/openapi.json

```diff
@@ -1126,6 +1126,54 @@
                             "type": "string"
                         }
                     },
+                    {
```

Comment:
Spec validator in CI raised this error:
~~~
Errors:
	- attribute paths.'/api/patch/v1/systems'(get).parameters.There are duplicate parameter
	  values

Error:  Spec has 1 errors.
Error: Process completed with exit code 1.
~~~

Reply:
oh, the problem is in handler annotation... see below.